crypto/engine/eng_lib.c

```diff
@@ -140,8 +140,10 @@ void engine_cleanup_add_first(ENGINE_CLEANUP_CB *cb)
     if (!int_cleanup_check(1))
         return;
     item = int_cleanup_item(cb);
-    if (item)
-        sk_ENGINE_CLEANUP_ITEM_insert(cleanup_stack, item, 0);
+    if (item != NULL) {
```

Comment:
Technically these curly brackets aren't needed either but we are more liberal about such cases.

Reply:
I did it similar to engine_cleanup_add_last.

Reply:
> Technically these curly brackets aren't needed either but we are more liberal about such cases.

Fixed